src/pages/Inventory.page.js

```diff
@@ -7,9 +7,41 @@ export class InventoryPage extends BaseSwagLabPage {
 
     inventoryItems = this.page.locator('.inventory_item');
 
+    inventoryItemNames = this.page.getByTestId('inventory-item-name');
+
+    inventoryItemPrices = this.page.getByTestId('inventory-item-price');
+
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
+    filterDropdown = this.page.getByTestId('product-sort-container');
+
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    async sortFromLowToHighPrice() {
```

Comment:
It might be better to create one method and switch/case inside the method

Reply:
changed